config.yaml

```diff
@@ -1,9 +1,9 @@
-group_id: test_group
-mytopics:
-- test_stream
-username: testuser
-password: null
-servers: localhost:9093
+fink_topics:
+  - test_stream
+fink_username: testuser
+fink_password: null
+fink_group_id: test_group
+fink_servers: localhost:9093
 skyportal_token: d47f6fb0-5b11-43fa-a725-7dbec3b8bd6b
```

Comment:
What is this token?

Reply:
I thought that an actual skyportal token as an example can help the user make sure that the token he copies and pastes (as instructed in the docs) is the right one. But maybe just a yaml comment saying where to find that token (like the actual path) would be better ?

Reply:
OK, as long as there is no security issue, it is fine to leave it like this. I would warn the user though that s.he needs to provide a valid token.